apps/ui/src/components/Ui/Button.vue

```diff
@@ -19,8 +19,12 @@ withDefaults(
   <button
     :type="type"
     :disabled="disabled || loading"
-    :class="primary && 'primary'"
-    class="rounded-full leading-[100%] border button px-3.5 h-[46px] text-skin-link bg-skin-bg"
+    :class="{
+      primary: primary,
+      'w-[46px] px-0': loading,
+      'px-3.5': !loading || $attrs.class?.includes('w-full')
```

Comment:
But previously we were adding `px-3.5` even if there is no `w-full` 🤔 it may effect some button size?

Reply:
So finally, did this work ?

Reply:
it works